pmd-core/src/test/java/net/sourceforge/pmd/cpd/CPDCommandLineInterfaceTest.java

```diff
@@ -22,8 +26,16 @@
 import net.sourceforge.pmd.PMD;
 import net.sourceforge.pmd.junit.JavaUtilLoggingRule;
 
+import com.google.common.collect.ImmutableMap;
+
 public class CPDCommandLineInterfaceTest {
     private static final String SRC_DIR = "src/test/resources/net/sourceforge/pmd/cpd/files/";
+    private static final Map<String, Integer> NUMBER_OF_TOKENS = ImmutableMap.of(
```

Comment:
That is a fair point. In this case, I had to adjust the test cases because they would break otherwise. Should we maybe move them to `XMLRendererTest`?

Reply:
Let's keep them here - the test the CLI interface of CPD, but also use the XML format.
I see, that you added a test in XMLRendererTest now, so it's fine with me.